### PR TITLE
Add `Ascaveth Invitational Tournament 2023` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
Wiki article for the recently concluded `Ascaveth Invitational Tournament 2023` LAN event.

Also including a fix for a minor grammatical error in `osu! Indonesia Tournament #13 (2023)`.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/AIT/2023/en.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+| Match 5U + Match 6U | 2023-12-09 (13:00 UTC +7) |
+| Match 7L + Match 8L | 2023-12-09 (14:15 UTC +7) |
+| Match 9L + Match 10L | 2023-12-09 (15:30 UTC +7) |
+| Match 11L + Match 12U | 2023-12-09 (16:45 UTC +7) |
+| *Break* | 2023-12-09 (18:00 UTC +7) |
+| Match 13L | 2023-12-09 (19:00 UTC +7) |
+| Match 14 (Grand Final) | 2023-12-09 (20:30 UTC +7) |
+| Awarding and Closing | 2023-12-09 (22:00 UTC +7) |
+
+For a visual representation of the structure of the matches, refer to [this chart](https://cdn.discordapp.com/attachments/548378597476532236/1186188416887558184/image.png?ex=6592570c&is=657fe20c&hm=35c8f49f33416d7097f54ec783e5bccdd8726b400c74049b388350de6884fee3&),
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:31]: put this image in the `img` folder

File: wiki/Tournaments/AIT/2023/en.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 35% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| *4th place* | 10% of the accumulated prize pool, exclusive players-only merch set |
+| *5th-8th place* | 5% of the accumulated prize pool, exclusive players-only merch set |
+
+As with other osu! tournaments in general, the Ascaveth Invitational Tournament 2023 was not  marked for profit, with all donations and proceeds going directly into the prize pool.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:45]:
```suggestion
As with other osu! tournaments in general, the Ascaveth Invitational Tournament 2023 was not marked for profit, with all donations and proceeds going directly into the prize pool.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+  2. [Hatsune Miku - Netoge Haijin Sprechchor (Lalarun) \[Insane\]](https://osu.ppy.sh/beatmapsets/25802#osu/87369)
+  3. [sakuzyo - Senkyou Ranbu (ktgster) \[Insane\]](https://osu.ppy.sh/beatmapsets/460516#osu/1042025)
+- Tiebreaker
+  1. **[Halozy - Sentimental Skyscraper (Hollow Wings) \[Myouren Hijiri\]](https://osu.ppy.sh/beatmapsets/100348#osu/412288)**  
+
+## Match results
+
+### Grand Finals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:213]:
```suggestion
| Bracket/match code | Player 1 |  |  | Player 2 | Match link |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower/Match 13L | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 1 | **7** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111641719) |
+| Grand Final/Match 14 | **[rHO](https://osu.ppy.sh/users/1629553)** ::{ flag=ID }:: | **7** | 6 | ::{ flag=ID }:: [DEETO](https://osu.ppy.sh/users/10069909) | [#1](https://osu.ppy.sh/community/matches/111642879) |
+
+### Finals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:220]:
```suggestion
| Bracket/match code | Player 1 |  |  | Player 2 | Match link |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower/Match 9L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 1 | ::{ flag=ID }:: [NazunaAmemiya](https://osu.ppy.sh/users/12159899) | [#1](https://osu.ppy.sh/community/matches/111639091) |
+| Lower/Match 10L | [Fuma](https://osu.ppy.sh/users/1501956) ::{ flag=ID }:: | 4 | **6** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111639131) |
+| Lower/Match 11L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 5 | ::{ flag=ID }:: [Hakui Koyori](https://osu.ppy.sh/users/10717635) | [#1](https://osu.ppy.sh/community/matches/111639900) |
+| Upper/Match 12U | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 3 | **6** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111639683) |
+
+### Semifinals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:229]:
```suggestion
| Bracket/match code | Player 1 |  |  | Player 2 | Match link |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 6]
```diff
@@ -0,0 +1,286 @@
...
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper/Match 5U | **[lifeline](https://osu.ppy.sh/users/11367222)** ::{ flag=ID }:: | **5** | 3 | ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956) | [#1](https://osu.ppy.sh/community/matches/111638406) |
+| Upper/Match 6U | [DEETO](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111638367) |
+| Lower/Match 7L | [Skydiver](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[NazunaAmemiya](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111637354) |
+| Lower/Match 8L | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 2 | **5** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111637200) |
+
+### Quarterfinals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:238]:
```suggestion
| Bracket/match code | Player 1 |  |  | Player 2 | Match link |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 7]
```diff
@@ -0,0 +1,286 @@
...
+| Upper/Match 2 | [NazunaAmemiya](https://osu.ppy.sh/users/12159899) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[Fuma](https://osu.ppy.sh/users/1501956)** | [#1](https://osu.ppy.sh/community/matches/111636028) |
+| Upper/Match 3 | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
+| Upper/Match 4 | [Hakui Koyori](https://osu.ppy.sh/users/10717635) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
+
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. All matches will be played over a single day in an undisclosed location following the Double Elimination system.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:250]:
```suggestion
2. All matches will be played over a single day in an undisclosed location following the double-elimination system.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 8]
```diff
@@ -0,0 +1,286 @@
...
+   - The bracket for the tournament will be randomly generated using [random.org](https://random.org) during the Technical Meeting.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:251]:
```suggestion
   - The bracket for the tournament will be randomly generated using [random.org](https://random.org) during the technical meeting.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 9]
```diff
@@ -0,0 +1,286 @@
...
+
+# Ascaveth Invitational Tournament 2023
+
+![Ascaveth Invitational Tournament 2023 logo](img/logo.png)
+
+The **Ascaveth Invitational Tournament 2023** was a double-elimination 1v1 invitational osu! tournament hosted by ::{ flag=ID }:: [Ascaveth](https://osu.ppy.sh/users/3245206) and co-organised by ::{ flag=ID }:: [Raids](https://osu.ppy.sh/users/15640966), ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805), ::{ flag=ID }:: [Sunohara Shun](https://osu.ppy.sh/users/3484548), and ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677). The tournament was the third major LAN osu! tournament to ever be held in Indonesia, as well as the second one to be run under the [Ascaveth Invitational Tournament](/wiki/Tournaments/AIT) banner.
+
+## Tournament schedule
+
+| Event | Timestamp |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:15]:
```suggestion
| Event | Timestamp (UTC+07) |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 10]
```diff
@@ -0,0 +1,286 @@
...
+| Match 3 + Match 4 | 2023-12-09 (10:30 UTC +7) |
+| *Break* | 2023-12-09 (11:30 UTC +7) |
+| Match 5U + Match 6U | 2023-12-09 (13:00 UTC +7) |
+| Match 7L + Match 8L | 2023-12-09 (14:15 UTC +7) |
+| Match 9L + Match 10L | 2023-12-09 (15:30 UTC +7) |
+| Match 11L + Match 12U | 2023-12-09 (16:45 UTC +7) |
+| *Break* | 2023-12-09 (18:00 UTC +7) |
+| Match 13L | 2023-12-09 (19:00 UTC +7) |
+| Match 14 (Grand Final) | 2023-12-09 (20:30 UTC +7) |
+| Awarding and Closing | 2023-12-09 (22:00 UTC +7) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:29]:
```suggestion
| Technical meeting | 2023-12-02 |
| Opening | 2023-12-09 (08:00) |
| Match 1 + match 2 | 2023-12-09 (09:00) |
| Match 3 + match 4 | 2023-12-09 (10:30) |
| *Break* | 2023-12-09 (11:30) |
| Match 5U + match 6U | 2023-12-09 (13:00) |
| Match 7L + match 8L | 2023-12-09 (14:15) |
| Match 9L + match 10L | 2023-12-09 (15:30) |
| Match 11L + match 12U | 2023-12-09 (16:45) |
| *Break* | 2023-12-09 (18:00) |
| Match 13L | 2023-12-09 (19:00) |
| Match 14 (Grand Final) | 2023-12-09 (20:30) |
| Awarding and closing | 2023-12-09 (22:00) |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 11]
```diff
@@ -0,0 +1,286 @@
...
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. All matches will be played over a single day in an undisclosed location following the Double Elimination system.
+   - The bracket for the tournament will be randomly generated using [random.org](https://random.org) during the Technical Meeting.
+3. All matches will be played using the setup provided by the tournament management on the spot.
+   - Players are allowed to bring in certain equipments (e.g. keyboard and graphics tablet) and their own device settings should they wish to.
+4. With the exception of the Grand Final tiebreaker, the maps and mappools for the tournament will be announced by the tournament management in advance before the event takes place.
+5. Match schedules will be predetermined by the tournament management.
+6. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:257]:
```suggestion
7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are awarded more fairly towards players who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 12]
```diff
@@ -0,0 +1,286 @@
...
+8. If a player disconnects, the referee will note the score the player had achieved at the end of the beatmap directly from their PC and adjust the outcome of the pick accordingly.
+9. Lag is not a valid reason to nullify a beatmap.
+10. If any problems during the match occur, the tournament management will make a decision based on the referee's report.
+11. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation. 
+    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the tournament management.
+    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+As the tournament was an invitational, there were no open registrations held for the main event. However, as always, testplayers, referees, and mappool selectors may not participate as players in the tournament.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:267]: random past tense

```suggestion
As the tournament is an invitational, there are no open registrations held for the main event. However, as always, testplayers, referees, and mappool selectors may not participate as players in the tournament.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 13]
```diff
@@ -0,0 +1,286 @@
...
+
+### Match instructions
+
+1. Each player must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
+   - The loser of the `!roll` gets the opposite by default.
+   - This rule does not apply in the Qualifier lobbies.
+2. Each player has to ban **one beatmap** (on the Quarterfinals and the Semifinals) or **two beatmaps** (on the Finals onwards) from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:275]:
```suggestion
2. Each player has to ban **one beatmap** (on the Quarterfinals and the Semifinals) or **two beatmaps** (from the Finals onwards) from the corresponding mappool. These beatmaps will not be allowed to be picked by any player during the entire match.
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 14]
```diff
@@ -0,0 +1,286 @@
...
+   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
+3. **There will be no warm-up beatmaps to be played in the multiplayer lobby.** Players who are looking up to warm themselves up before the match are expected to do so by their own during the grace period before the match commences.
+4. **There will be no FreeMod pick in the mappool.** However, in the case of a tiebreaker, the tiebreaker map will be played with the FreeMod option enabled which allows players to play the tiebreaker map with EZ, HR, HD, or any possible combinations of the three mods should they wish to.
+   - Playing the tiebreaker map with a mod is *not* mandatory.
+5. Players who can compete in the next round are determined by:
+   - In the Quarterfinals and the Semifinals, each player needs to win 5 points in order to win a match. (Best-of-9)
+   - In the Finals, each player needs to win 6 points in order to win a match. (Best-of-11)
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:282]:
```suggestion
   - In the Quarterfinals and the Semifinals, each player needs to win 5 points in order to win a match. (Best of 9)
   - In the Finals, each player needs to win 6 points in order to win a match. (Best of 11)
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 15]
```diff
@@ -0,0 +1,286 @@
...
+  1. **[Halozy - Sentimental Skyscraper (Hollow Wings) \[Myouren Hijiri\]](https://osu.ppy.sh/beatmapsets/100348#osu/412288)**  
+
+## Match results
+
+### Grand Finals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower/Match 13L | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 1 | **7** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111641719) |
+| Grand Final/Match 14 | **[rHO](https://osu.ppy.sh/users/1629553)** ::{ flag=ID }:: | **7** | 6 | ::{ flag=ID }:: [DEETO](https://osu.ppy.sh/users/10069909) | [#1](https://osu.ppy.sh/community/matches/111642879) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:216]:
```suggestion
| Lower/match 13L | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 1 | **7** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111641719) |
| Grand Final/match 14 | **[rHO](https://osu.ppy.sh/users/1629553)** ::{ flag=ID }:: | **7** | 6 | ::{ flag=ID }:: [DEETO](https://osu.ppy.sh/users/10069909) | [#1](https://osu.ppy.sh/community/matches/111642879) |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 16]
```diff
@@ -0,0 +1,286 @@
...
+
+### Finals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower/Match 9L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 1 | ::{ flag=ID }:: [NazunaAmemiya](https://osu.ppy.sh/users/12159899) | [#1](https://osu.ppy.sh/community/matches/111639091) |
+| Lower/Match 10L | [Fuma](https://osu.ppy.sh/users/1501956) ::{ flag=ID }:: | 4 | **6** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111639131) |
+| Lower/Match 11L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 5 | ::{ flag=ID }:: [Hakui Koyori](https://osu.ppy.sh/users/10717635) | [#1](https://osu.ppy.sh/community/matches/111639900) |
+| Upper/Match 12U | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 3 | **6** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111639683) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:225]:
```suggestion
| Lower/match 9L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 1 | ::{ flag=ID }:: [NazunaAmemiya](https://osu.ppy.sh/users/12159899) | [#1](https://osu.ppy.sh/community/matches/111639091) |
| Lower/match 10L | [Fuma](https://osu.ppy.sh/users/1501956) ::{ flag=ID }:: | 4 | **6** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111639131) |
| Lower/match 11L | **[DEETO](https://osu.ppy.sh/users/10069909)** ::{ flag=ID }:: | **6** | 5 | ::{ flag=ID }:: [Hakui Koyori](https://osu.ppy.sh/users/10717635) | [#1](https://osu.ppy.sh/community/matches/111639900) |
| Upper/match 12U | [lifeline](https://osu.ppy.sh/users/11367222) ::{ flag=ID }:: | 3 | **6** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111639683) |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 17]
```diff
@@ -0,0 +1,286 @@
...
+
+### Semifinals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper/Match 5U | **[lifeline](https://osu.ppy.sh/users/11367222)** ::{ flag=ID }:: | **5** | 3 | ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956) | [#1](https://osu.ppy.sh/community/matches/111638406) |
+| Upper/Match 6U | [DEETO](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111638367) |
+| Lower/Match 7L | [Skydiver](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[NazunaAmemiya](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111637354) |
+| Lower/Match 8L | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 2 | **5** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111637200) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:234]:
```suggestion
| Upper/match 5U | **[lifeline](https://osu.ppy.sh/users/11367222)** ::{ flag=ID }:: | **5** | 3 | ::{ flag=ID }:: [Fuma](https://osu.ppy.sh/users/1501956) | [#1](https://osu.ppy.sh/community/matches/111638406) |
| Upper/match6U | [DEETO](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111638367) |
| Lower/match 7L | [Skydiver](https://osu.ppy.sh/users/10069909) ::{ flag=ID }:: | 4 | **5** | ::{ flag=ID }:: **[NazunaAmemiya](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111637354) |
| Lower/match 8L | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 2 | **5** | ::{ flag=ID }:: **[Hakui Koyori](https://osu.ppy.sh/users/10717635)** | [#1](https://osu.ppy.sh/community/matches/111637200) |
```

File: wiki/Tournaments/AIT/2023/en.md [hunk 18]
```diff
@@ -0,0 +1,286 @@
...
+
+### Quarterfinals
+
+| Bracket/Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper/Match 1 | **[lifeline](https://osu.ppy.sh/users/11367222)** ::{ flag=ID }:: | **5** | 2 | ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008) | [#1](https://osu.ppy.sh/community/matches/111636056) |
+| Upper/Match 2 | [NazunaAmemiya](https://osu.ppy.sh/users/12159899) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[Fuma](https://osu.ppy.sh/users/1501956)** | [#1](https://osu.ppy.sh/community/matches/111636028) |
+| Upper/Match 3 | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
+| Upper/Match 4 | [Hakui Koyori](https://osu.ppy.sh/users/10717635) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:243]:
```suggestion
| Upper/match 1 | **[lifeline](https://osu.ppy.sh/users/11367222)** ::{ flag=ID }:: | **5** | 2 | ::{ flag=ID }:: [Skydiver](https://osu.ppy.sh/users/4750008) | [#1](https://osu.ppy.sh/community/matches/111636056) |
| Upper/match 2 | [NazunaAmemiya](https://osu.ppy.sh/users/12159899) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[Fuma](https://osu.ppy.sh/users/1501956)** | [#1](https://osu.ppy.sh/community/matches/111636028) |
| Upper/match 3 | [ThatNOOBGuy](https://osu.ppy.sh/users/11091594) ::{ flag=ID }:: | 1 | **5** | ::{ flag=ID }:: **[DEETO](https://osu.ppy.sh/users/10069909)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
| Upper/match 4 | [Hakui Koyori](https://osu.ppy.sh/users/10717635) ::{ flag=ID }:: | 0 | **5** | ::{ flag=ID }:: **[rHO](https://osu.ppy.sh/users/1629553)** | [#1](https://osu.ppy.sh/community/matches/111636056) |
```

File: wiki/Tournaments/AIT/2023/en.md (hunk 7 above)
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:250]: who are these people taking osu! players to random undisclosed locations
i assume it was disclosed to invitees

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/AIT/2023/en.md [hunk 19]
```diff
@@ -0,0 +1,286 @@
...
+
+The Ascaveth Invitational Tournament 2023 offered a prize pool that was built upon community donations through [Saweria](https://saweria.co/osuIndonesia) and [Streamlabs](https://streamlabs.com/osuindonesia/tip) as well as [merchandise sales](https://www.youtube.com/watch?v=3J8BuScGRSY).
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 35% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 20% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the accumulated prize pool, physical trophy (courtesy of [Wellen Print](https://wellenprint.com/)), commemorative medal, exclusive players-only merch set |
+| *4th place* | 10% of the accumulated prize pool, exclusive players-only merch set |
+| *5th-8th place* | 5% of the accumulated prize pool, exclusive players-only merch set |
```
Comment (Walavouchey) [inline wiki/Tournaments/AIT/2023/en.md:43]:
```suggestion
| *5th–8th place* | 5% of the accumulated prize pool, exclusive players-only merch set |
```

File: wiki/Tournaments/AIT/2023/en.md (hunk 7 above)
Comment (Nivalyx) [reply wiki/Tournaments/AIT/2023/en.md]: yeah, due to space and logistical concerns the entire thing was only open to the staff members and the invitees

do you have something in mind about how this should be phrased tho? i dunno, i think the current phrasing is already clear enough in my opinion but yea as always feel free to throw in a suggestion if you have something here

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

File: wiki/Tournaments/AIT/2023/en.md (hunk 7 above)
Comment (Walavouchey) [reply wiki/Tournaments/AIT/2023/en.md]: adjusted in 4e4255c101d56d07e47442609b34405fd52dedd3

**Pull request merged**